src/borg/remote.py

```diff
@@ -636,6 +636,12 @@ def borg_cmd(self, args, testing):
                 opts.append('--critical')
             else:
                 raise ValueError('log level missing, fix this code')
+            try:
+                borg_logger = logging.getLogger('borg')
+                if borg_logger.json:
+                    opts.append('--log-json')
```

Comment:
so, you give --log-json to the remote borg serve, but in logger:95, you do not use the JsonFormatter if it is in the borg serve process? so, what is it good for?